css/styles.css

```diff
@@ -8,3 +19,707 @@ body {
   font-family: Arial, Helvetica, sans-serif;
   overflow-x: hidden;
 }
+
+ol,
+ul {
+  list-style: none;
+  margin: 0;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+h2,
+h3,
+h4,
+h5,
+h6,
+.h2,
+.h3,
+.h4 {
+  font-family: "Roboto Condensed", sans-serif;
+  margin-bottom: 10px;
+  position: relative;
+  color: black;
+  font-weight: bold;
+}
+
+footer .footer-logo {
+  fill: red;
+  font-size: 2rem;
+  height: 3.3rem;
+}
+
+footer i {
+  color: gray;
+  padding: 0.6rem;
+  font-size: 2.2rem;
+}
+
+.copy-right-footer {
+  font-size: 13px;
+}
+
+footer .editions-links li,
+footer .about-links li {
+  padding: 0 1rem;
+  border-left: 1px solid rgb(66, 65, 65);
+}
+
+footer .about-links li {
+  font-size: 14px;
+}
+
+/* navbar */
+
+.mainNav {
+  background-color: #f72210;
+  min-height: 50px;
+}
+
+.logoNav {
+  width: 130px;
+  height: 20px;
+  fill: white;
+}
+
+.rightBtns {
+  margin: 0;
+}
+
+.logIn {
+  font-size: 1.25em;
+  margin-right: 1em;
+  font-weight: bold;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.sub {
+  background-color: #ffe400;
+  border-radius: 24px;
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+  padding: 0 1em;
+  margin: 0 0.5em;
+}
+
+.sub::after {
+  content: "";
+  display: inline-block;
+  margin-left: 3px;
+  width: 4px;
+  height: 4px;
+  transform: rotate(45deg);
+  border-top-width: 1px;
+  border-top-style: solid;
+  border-right-width: 1px;
+  border-right-style: solid;
+  border-color: black;
+}
+
+.burger1 {
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex-direction: column;
+  flex-direction: column;
+  -webkit-align-items: center;
+  align-items: center;
+  -webkit-justify-content: center;
+  justify-content: center;
+  width: 30px;
+  height: 30px;
+  vertical-align: top;
+  text-align: center;
+}
```

Comment:
- [ ]  A lot of the styles here can be achieved with Bootstrap.
Try to go through your CSS styles and use Boostrap classes for the ones that are available instead.

Example of styles that can be done with Bootstrap classes:
- width 100%
- display properties
- position properties
- all flex and flex justify/align related properties
- border 0
- margin 0
- padding 0
- text align
- font weight

Check in the [Bootstrap documentation](https://getbootstrap.com/docs/4.0/components/navs/) the classes you can use in your project.